core/src/main/java/org/elasticsearch/cluster/metadata/MetaDataMappingService.java

```diff
@@ -320,28 +319,24 @@ private ClusterState applyRequest(ClusterState currentState, PutMappingClusterSt
 
                     }
                 } else {
```

Comment:
It is logging only indeed. But whether or not to log things tends to be a bit controversial so I'd rather do it in another change if we want to remove it